### PR TITLE
chore: Prep changelog for release

### PR DESCRIPTION
**Pull request by spencergilbert**
Signed-off-by: Spencer Gilbert <spencer.gilbert@datadoghq.com>

### CODE REVIEW

**Review by jszwedko — APPROVED** [review]
🚀 

Should the release notes also include that the version of Vector was updated?

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,26 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.1.0-beta.0] - 2021-11-02
+
+### Vector
+
+#### Bug Fixes
+
+- Fix vector proto in haproxy config (#94) ([b0a7d7d](https://github.com/vectordotdev/helm-charts/commit/b0a7d7d6e430c93376e498ce1653ac9a331d5e60))
```
Comment (StephenWakely) [inline CHANGELOG.md:12]: What is a proto?

Comment (tobz) [reply CHANGELOG.md:12]: The protocol being used/that should be used by HAProxy for traffic.  In this case, changing it from the default of HTTP to HTTP/2, to align with the `vector` v2 source/sink using gRPC now.

**Review by tobz — APPROVED** [review]
LGTM.

**Pull request merged**